stubs/networkx/networkx/algorithms/clique.pyi

```diff
@@ -1,17 +1,20 @@
+from _typeshed import Unused
 from collections.abc import Iterable
 
 from networkx.classes.graph import Graph, _Node
 
 def enumerate_all_cliques(G: Graph[_Node]) -> Iterable[list[_Node]]: ...
 def find_cliques(G: Graph[_Node], nodes: list[_Node] | None = None) -> Iterable[list[_Node]]: ...
 def find_cliques_recursive(G: Graph[_Node], nodes: list[_Node] | None = None) -> Iterable[list[_Node]]: ...
-def make_max_clique_graph(G: Graph[_Node], create_using: type[Graph[_Node]] = None) -> Graph[_Node]: ...
+def make_max_clique_graph(G: Graph[_Node], create_using: type[Graph[_Node]] | None = None) -> Graph[_Node]: ...
```

Comment:
I would have sworn that I tried applying a patch exactly like this, except that mypy yelled at me about unsolvable types...

regardless, this does not quite express the correct thing, which is to link the types of create_using and the return type (the higher-kinded example I gave before). I'll fix it.